clang/tools/clang-sycl-linker/ClangSYCLLinker.cpp

```diff
@@ -349,13 +351,55 @@ Error runSYCLLink(ArrayRef<std::string> Files, const ArgList &Args) {
   if (!LinkedFile)
     reportError(LinkedFile.takeError());
 
+  // TODO: SYCL post link functionality involves device code splitting and will
```

Comment:
code splitting PR is under review here: https://github.com/llvm/llvm-project/pull/131347

Thanks